pilot/pkg/proxy/envoy/v1/authentication.go

```diff
@@ -71,7 +71,7 @@ func buildJwksURIClustersForProxyInstances(mesh *meshconfig.MeshConfig,
 	}
 	var jwtSpecs []*authn.Jwt
 	for _, instance := range proxyInstances {
-		authnPolicy := model.GetConsolidateAuthenticationPolicy(mesh, store, instance.Service.Hostname, instance.Endpoint.ServicePort)
+		authnPolicy := model.GetConsolidateAuthenticationPolicy(mesh, store, instance.Service.Hostname, instance.Endpoint.ServicePort, instance.Labels)
 		jwtSpecs = append(jwtSpecs, authn_plugin.CollectJwtSpecs(authnPolicy)...)
```

Comment:
There is no destination rule for v1. So how will the comparison work ?

Reply:
Since v1 is going be deprecated soon, policy dark launch won't support v1(again, will be confirmed after design review).